### PR TITLE
Link ebiP2TitleFog.cpp

### PR DESCRIPTION
**Pull request by HeartPiece44**
Links ebiP2TitleFog.cpp and fixes related headers.

### CODE REVIEW

**Comment by jonathonbrady** [comment]
did you run clang-format?

**Review by jonathonbrady — APPROVED** [review]
lgtm

File: include/ebi/title/TTitleFogMgr.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#include "Camera.h"
+#include "Parameters.h"
+#include "BaseParm.h"
+
+namespace ebi {
+namespace title {
+/**
+ * @size{0x140}
+ */
+struct TTitleFogMgr : public FogMgr {
```
Comment (EpochFlame) [inline include/ebi/title/TTitleFogMgr.h:16]: what the heck is this?

Comment (EpochFlame) [reply include/ebi/title/TTitleFogMgr.h:16]: I should have known. This is how parms work across the repo 🤦

**Review by EpochFlame — APPROVED** [review]
good job hp, bless

**Pull request merged**